dle/data/migrations/0001_initial 2.py

```diff
@@ -0,0 +1,99 @@
+# Generated by Django 4.0.2 on 2022-04-17 23:06
```

Comment:
What's going on with the migrations here? Is this a change you made or is this somehow getting in here by accident?